<launch>

    <include file="$(find eva_a)/launch/set_gas_parameters.launch"/>
    
    <include file="$(find eva_a)/launch/turtlebot3_eva_b.launch"/>
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
    <include file="$(find turtlebot3_navigation)/launch/turtlebot3_navigation.launch">
	  <arg name="map_file" value="$(find eva_a)/map/map.yaml" />
    </include>

    <node pkg="tf2_web_republisher" name="tf2_web_republisher" type="tf2_web_republisher"/>
    <node pkg="eva_a" name="gas_level" type="gas_level_publisher.py"/>
    <node pkg="eva_a" name="gas_sensor_monitoring" type="gas_sensor_monitoring.py"/>

    <param name="/amcl/initial_pose_x" value="1.0" />
    <param name="/amcl/initial_pose_y" value="-3.0" />
    
</launch>
